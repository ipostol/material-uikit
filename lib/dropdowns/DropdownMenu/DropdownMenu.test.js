'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _enzyme = require('enzyme');

var _Option = require('../Option');

var _Option2 = _interopRequireDefault(_Option);

var _DropdownMenu = require('./DropdownMenu');

var _DropdownMenu2 = _interopRequireDefault(_DropdownMenu);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

describe('DropdownMenu component', function () {

  it('should render correctly', function () {

    var props = {
      options: [{ id: 1 }]
    };

    var wrapper = (0, _enzyme.shallow)(_react2.default.createElement(_DropdownMenu2.default, props));

    wrapper.find(_Option2.default).simulate('click');

    expect(wrapper).toMatchSnapshot();
  });
});