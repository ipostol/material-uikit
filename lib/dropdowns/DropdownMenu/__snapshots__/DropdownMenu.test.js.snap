// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownMenu component should render correctly 1`] = `
<div
  className="container"
>
  <Dropdown
    trigger={
      <div
        className="trigger"
      >
        <div
          className="icon"
        >
          <ArrowDown />
        </div>
      </div>
    }
  >
    <Option
      element="div"
      onClick={[Function]}
      size="s"
    />
  </Dropdown>
</div>
`;
